app/main.py

```diff
@@ -1,8 +1,21 @@
 class Person:
-    # write your code here
-    pass
+    people = dict()
+
+    def __init__(self,
+                 name: str,
+                 age: int) -> None:
```

Comment:
It is a common practice to place each argument on a new line in the `__init__` so that opening and closing parenthesis would not have any arguments on the same line. Please, follow this approach.